### ISSUE TITLE
Centos stream hosts are not added to cluster

### ISSUE THREAD

**Issue by iqfx** [mrlesmithjr/ansible-mariadb-galera-cluster#73]
**Describe the bug**
After running the role, the nodes are not added to the cluster. No error messages are shown

**To Reproduce**
run the role on centos stream 8 with three nodes on the same network. (x86)

vars used:
galera_cluster_nodes_group: "database"
mariadb_version: "10.5"
galera_cluster_bind_interface: "eth1"
galera_cluster_name: "db-cluster"
galera_reconfigure_galera: true
mariadb_databases:
  - name: monitoring

**Expected behavior**
The nodes should be added to the galera cluster

**Screenshots**
This is the output I get from this role
![image](https://user-images.githubusercontent.com/39005172/119670158-f3a42900-be38-11eb-9c5a-e4ad4dd4f7ac.png)

Also it skips ALOT of tasks

![image](https://user-images.githubusercontent.com/39005172/119670281-0d457080-be39-11eb-8702-90cc2eb27d68.png)

On all nodes after deployment it shows the following:

```
MariaDB [(none)]> SHOW GLOBAL STATUS LIKE 'wsrep_%';
+-------------------------------+----------------------+
| Variable_name                 | Value                |
+-------------------------------+----------------------+
| wsrep_applier_thread_count    | 0                    |
| wsrep_cluster_capabilities    |                      |
| wsrep_cluster_conf_id         | 18446744073709551615 |
| wsrep_cluster_size            | 0                    |
| wsrep_cluster_state_uuid      |                      |
| wsrep_cluster_status          | Disconnected         |
| wsrep_connected               | OFF                  |
| wsrep_local_bf_aborts         | 0                    |
| wsrep_local_index             | 18446744073709551615 |
| wsrep_provider_capabilities   |                      |
| wsrep_provider_name           |                      |
| wsrep_provider_vendor         |                      |
| wsrep_provider_version        |                      |
| wsrep_ready                   | OFF                  |
| wsrep_rollbacker_thread_count | 0                    |
| wsrep_thread_count            | 0                    |
+-------------------------------+----------------------+
```

**Additional context**
I have tried enabling the reconfigure cluster variable. Produces the output shown above. The installs of mariadb and galera are NOT fresh. Fresh installs seem to work fine.

**Comment by iqfx** [comment]
Using version 0.3.1

**Comment by mrlesmithjr** [comment]
Maybe @eRadical has some insight? ;)

**Comment by eRadical** [comment]
It just happens that I have to start a cluster... 
I'll migrate to CentOS 8 stream before running the galera role over the machines.

I'll get back soon!

**Comment by mrlesmithjr** [comment]
Awesome timing!

**Comment by eRadical** [comment]
Duplicate of #71.

Will be resolved w/ #72.

**Comment by eRadical** [comment]
I started 3 Centos8 machines (in AWS) and switched them to CentOS Stream (as described in https://www.centos.org/centos-stream/).
After this I applied the role w/ the changes from #72  and it worked.

Did not test `reconfigure`.

**Comment by iqfx** [comment]
I really feel like this is a different issue as described in the merge request above. I cloned your branch and result stays the same :/ Thanks alot for all of the help so far though! <3

**Comment by eRadical** [comment]
Did you by any chance changed the `sst` to `mariabackup`?

If yes: Can you look in the first node and see if the password for mariabackup is correct (start it by hand if it's not up).

**Comment by iqfx** [comment]
I found out why the role is not working on my machines, turns out when running a triple master setup, and rebooting all three 3 nodes at the same time (I know probably something you don't want to do anyway). It ruines the entire cluster and I have yet to find a way for resetting mariadb and galera completely. I have removed packages mariadb-server, and galera-4, then removed dirs /var/lib/mysql and /etc/my.cnf.d/ but that does not seem to be enough even after a reboot. This issue is therefore a complete edgecase. My apologies!
